planning/obstacle_velocity_limiter/src/obstacle_velocity_limiter_node.cpp

```diff
@@ -80,6 +79,18 @@ ObstacleVelocityLimiterNode::ObstacleVelocityLimiterNode(const rclcpp::NodeOptio
 
   set_param_res_ =
     add_on_set_parameters_callback([this](const auto & params) { return onParameter(params); });
+
+  // transform
+  rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+  map_odom_tf_buffer_ = std::make_shared<tf2_ros::Buffer>(clock);
+  map_odom_tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*map_odom_tf_buffer_);
+
+  // transformed msg publishers
+  occupancy_grid_pub_ = create_publisher<OccupancyGrid>(
+          "/planning/scenario_planning/lane_driving/motion_planning/occupancy_grid_map/map", 1);
+
+  dynamic_obstacles_pub_ = create_publisher<PredictedObjects>(
+          "/planning/scenario_planning/lane_driving/motion_planning/predicted_objects", 1);
```

Comment:
Please use the same format as other topics and remap.

Reply:
> Please use the same format as other topics and remap.

fixed. thank you.